src/main/views/cookies.njk

```diff
@@ -0,0 +1,74 @@
+{% extends "layout.njk" %}
+{% from "support.njk" import supportWidget %}
+{% set heading = 'Cookies' %}
+
+{% set title = 'Money claim' %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+
+      <p>
+        {{ t('A cookie is a small piece of data that’s stored on your computer, '
+        + 'tablet, or phone when you visit a website. '
+        + 'Most websites need cookies to work properly.') }}
+      </p>
+
+      <p> {{ t('This service uses them to:') }}</p>
+
+      <ul class="list list-bullet">
+        <li>{{ t('measure how you use the service so it can be updated and improved based on your needs') }}</li>
+        <li>{{ t('remember the notifications you’ve seen so that you’re not shown them again') }}</li>
+        <li>{{ t('temporarily store the answers you give') }}</li>
+      </ul>
+
+      <p>
+        {{ t('Find out more about how to <a href="{{ cookieInformation }}">manage cookies</a>.',
+        { cookieInformation : 'https://www.aboutcookies.org'}) | safe }}
+      </p>
+
+      <h2 class="heading-medium">{{ t('How cookies are used in the Civil money claims') }}</h2>
+      <h2 class="heading-small">{{ t('To measure website usage') }}</h2>
+
+      <p>
+        {{ t('We use Google Analytics software to collect information about how you use this service.'
+        + 'We do this to help make sure the service is meeting the needs of its users'
+        +' and to help us make improvements, for example improving site search.') }}
+      </p>
+
+      <p> {{ t('Google Analytics stores information about:') }}</p>
+
+      <ul class="list list-bullet">
+        <li>{{ t('the pages you visit') }}</li>
+        <li>{{ t('how long you spend on each page') }}</li>
+        <li>{{ t('how you got to the service') }}</li>
+        <li>{{ t('what you click on while you’re visiting the service') }}</li>
+      </ul>
+
+      <p>
+        {{ t('We allow Google to use or share our analytics data. You can find out more '
+        + 'about how Google use this information in their <a href="{{ privacyPolicy }}">privacy policy</a>.',
```

Comment:
Please use `AppPaths` in the template directly. That will make future refactoring or tracing links easier if we have single way of doing it.

Reply:
👍